apps/prairielearn/src/pages/administratorSettings/administratorSettings.html.ts

```diff
@@ -88,6 +88,16 @@ export function AdministratorSettings({ resLocals }) {
               </script>
             </div>
           </div>
+          <div class="card mb-4">
```

Comment:
This should also have an `isEnterprise()` guard.

Reply:
Done in 99f9ce0e0871e9b04042c1180fd183dcf2f45434.